scripts/play.ts

```diff
@@ -0,0 +1,380 @@
+
+    await createPlayground(playgroundName, playgroundRoot, {
+      framework: args.framework ?? DEFAULT_FRAMEWORK,
+      dryRun: args["dry-run"],
+    });
+  }
+
+  if (args.start) {
+    await exec("yarn", ["dev"], {
+      cwd: playgroundRoot,
+      stdio: "inherit",
+      dryRun: args["dry-run"],
+    });
+  }
+}
+
+/**
+ * Creates a new playground.
+ *
+ * @param name - The name of the playground. It must be unique.
+ * @param root - The directory of the playground. It must be unique.
+ * @param options - Options that determine the function's behavior.
+ */
+async function createPlayground(
+  name: string,
+  root: string,
+  options: {
+    /**
+     * The framework used to bootstrap the playground.
+     */
+    framework: Args["framework"];
+
+    /**
+     * If `true`, commands are not executed.
+     */
+    dryRun?: boolean;
+  },
+) {
+  switch (options.framework ?? "next") {
+    case "next": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/nextjs-starter-prismic-minimal-ts.git",
+        root,
+        { dryRun: options.dryRun },
+      );
+
+      await updateJSONFile<PackageJSON>(
+        path.join(root, "package.json"),
+        { scripts: { "dev:website": "next dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    case "sveltekit": {
+      await cloneGitRepo(
+        "https://github.com/prismicio-community/sveltekit-starter-prismic-minimal.git",
+        root,
+        { dryRun: options.dryRun },
+      );
+
+      await updateJSONFile<PackageJSON>(
+        path.join(root, "package.json"),
+        { scripts: { "dev:website": "vite dev --port=8001" } },
+        { dryRun: options.dryRun },
+      );
+
+      break;
+    }
+
+    default: {
+      console.error(chalk.red`Framework not supported: ${options.framework}`);
+      process.exit(1);
+    }
+  }
+
+  if (!options.dryRun) {
+    await fs.rm(path.join(root, ".git"), { recursive: true });
+    await fs.rm(path.join(root, "package-lock.json"));
+  }
+
+  await exec("yarn", ["add", "--dev", "concurrently"], {
+    cwd: root,
+    stdio: "inherit",
+    dryRun: options.dryRun,
+  });
+
+  await updateJSONFile<PackageJSON>(
+    path.join(root, "package.json"),
+    {
+      name,
+      scripts: {
+        dev: 'concurrently --prefix-colors auto "yarn:dev:website" "yarn:dev:slicemachine"',
+        "dev:slicemachine": `SM_ENV=staging ../../packages/start-slicemachine/bin/start-slicemachine.js`,
+        slicemachine:
+          "../../packages/start-slicemachine/bin/start-slicemachine.js",
+      },
+    },
+    { dryRun: options.dryRun },
+  );
+
+  await updateJSONFile<SliceMachineConfig>(
+    path.join(root, "slicemachine.config.json"),
+    {
+      localSliceSimulatorURL: "http://localhost:8001/slice-simulator",
+      apiEndpoint: `https://${name}.cdn.wroom.io/api/v2`,
+    },
+    { dryRun: options.dryRun },
+  );
+
+  await exec(
+    "npx",
```

Comment:
It does work, but it takes noticeably longer since `yarn dlx` doesn't cache its installations like `npx` does. It's not too bad though. Let's go with `yarn dlx` and if we get annoyed with the creation time, we have ways to optimize it.

Reply:
I had to change it back because `yarn dlx` was running into a dependency issue with `readable-web-to-node-stream`, a dependency of `file-type`, which we use in `@slicemachine/manager`. It's weird though: I didn't see it when I first tried it.

I reverted back to `npx` because it works reliably, but we will need to find a proper fix.